.travis.yml

```diff
@@ -0,0 +1,23 @@
+language: python
+
+python:
+  - 3.7
+
+install:
+  - cd Project/website
+  - pip install -r requirements.txt
+
+before_script:
+  - python manage.py migrate
+  #- python manage.py collectstatic --noinput
```

Comment:
Was this line particularly buggy? 
Info - Collectstatic is a command which collects all your assets and places them in the "staticfiles" directory defined in your settings. This is particularly helpful when deploying your website to a server.

Reply:
Oh I thought the configurations in the settings.py file already does that. Yeah, this line threw out an error when I ran it.

Reply:
The error might've been something on the lines of "static_root" not defined. Essentially, you would have to assign a value to the "static_root" variable in your settings file - we usually use the folder name "staticfiles" to differentiate it from "static".
STATIC_ROOT = os.path.join(BASE_DIR, "staticfiles") should do the trick for you. 
Additionally, check this description for more info - [link](https://docs.djangoproject.com/en/3.0/ref/contrib/staticfiles/#collectstatic)

Reply:
It works :D